src/main/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1.java

```diff
@@ -281,21 +277,24 @@ public ResponseEntity<Integer> memberAttributeResultsAsync(
     }
 
     /**
-     * Get all the members of an owned grouping through paginated calls.
-     */
+     * Get all the members of an owned grouping through paginated calls. This should be done through the UI using await.
+     * Currently, the UI is not using this function to hydrate grouping members, this implementation of getMembers is
+     * much faster than the current getMembers in use and should be used in the future ether when GROUPINGS-304 is
+     * completed on the UI or when the UI is migrated to the react framework.
+ */
     @PostMapping(value = "/groupings/group")
     @ResponseBody
-    public ResponseEntity<GroupingGroupsMembers> ownedGrouping(@RequestHeader(CURRENT_USER_KEY) String currentUser,
+    public ResponseEntity<GroupingGroupsMembers> getGroupingMembers(@RequestHeader(CURRENT_USER_KEY) String currentUser,
                                                                @RequestBody List<String> groupPaths,
-                                                               @RequestParam Integer page,
-                                                               @RequestParam Integer size,
-                                                               @RequestParam String sortString,
-                                                               @RequestParam Boolean isAscending) {
-        logger.info("Entered REST ownedGrouping...");
+                                                               @RequestParam(required = true) Integer page,
+                                                               @RequestParam(required = true) Integer size,
+                                                               @RequestParam(required = true) SortBy sortBy,
+                                                               @RequestParam(required = true) Boolean isAscending) {
+        logger.info("Entered REST getGrouping...");
```

Comment:
Could you revert all the changes back to the old diff but of course keep the SortBy change.

Basically, change the javadoc comment back, rename getGroupingMembers to ownedGrouping, including the logger.info, and take off of (required = true)? Thanks.